Library not compiling on Windows

Path not found for Windows

`In file included from D:\Leo\Documents\Arduino\libraries\107-Arduino-MCP2515\src/ArduinoMCP2515.h:16:0,

Несколько библиотек найдено для "ArduinoMCP2515.h"
                 from D:\Leo\Documents\Arduino\sketch_apr25a\sketch_apr25a.ino:2:

 Используется: D:\Leo\Documents\Arduino\libraries\107-Arduino-MCP2515
D:\Leo\Documents\Arduino\libraries\107-Arduino-MCP2515\src/MCP2515/MCP2515_Io.h:17:10: fatal error: functional: No such file or directory`

but library exist in 
D:\Leo\Documents\Arduino\libraries\107-Arduino-MCP2515

Arduino: 1.8.12 (Windows 8.1), Плата:"Arduino Uno"

In file included from D:\Leo\Documents\Arduino\libraries\107-Arduino-MCP2515\src/ArduinoMCP2515.h:16:0,

                 from D:\Leo\Documents\Arduino\sketch_apr25a\sketch_apr25a.ino:2:

D:\Leo\Documents\Arduino\libraries\107-Arduino-MCP2515\src/MCP2515/MCP2515_Io.h:17:10: fatal error: functional: No such file or directory

 #include <functional>

          ^~~~~~~~~~~~

compilation terminated.

exit status 1
Ошибка компиляции для платы Arduino Uno.

Этот отчёт будет иметь больше информации с
включенной опцией Файл -> Настройки ->
"Показать подробный вывод во время компиляции"

--- a/library.properties
+++ b/library.properties
@@ -6,5 +6,5 @@ sentence=Arduino library for controlling the MCP2515 in order to receive/transmi
 paragraph=
 category=Communication
 url=https://github.com/107-systems/107-Arduino-MCP2515
-architectures=*
+architectures=samd
 includes=ArduinoMCP2515.h

